lsif/src/dump-manager/settings.ts

```diff
@@ -18,3 +18,15 @@ export const DOCUMENT_CACHE_CAPACITY = readEnvInt('DOCUMENT_CACHE_CAPACITY', 102
 
 /** The maximum number of result chunks that can be held in memory at once. */
 export const RESULT_CHUNK_CACHE_CAPACITY = readEnvInt('RESULT_CHUNK_CACHE_CAPACITY', 1024 * 1024 * 1024)
+
+/** The maximum rate that the server will send upload payloads. */
+export const MAXIMUM_SERVE_BYTES_PER_SECOND = readEnvInt('MAXIMUM_SERVE_BYTES_PER_SECOND', 1024 * 1024 * 1024 * 10) // 10 Gib/sec
```

Comment:
```suggestion
export const MAXIMUM_SERVE_BYTES_PER_SECOND = readEnvInt('MAXIMUM_SERVE_BYTES_PER_SECOND', 1024 * 1024 * 1024 * 10) // 10 GiB/sec
```

(now it's actually bytes)

This throttler is global, but per-process, yes? I am still skeptical we will see a container network ever reach this cap even fully saturated. That's not a blocker, just seems weird to have a default that's effectively "unbounded".

Reply:
cc @slimsag for a more sensible default suggestion